platform-coop-toolkit/pinecone
# fix: resolve issue with nested checkboxes (resolves #253)

## Description

Fixes #253.

## Steps to test

From the Archive layout, check the top level 'Cooperative essentials' checkbox, then 'Apply Filters'.

**Expected Behavior**

Query string should show the top level topic and all sub topics with no empty query values.

## Additional information

Not applicable.

## Related issues

- Resolves #253.
